src/cli/extensions/helpers.js

```diff
@@ -27,6 +30,8 @@ export function runPostInits(initialState) {
 
 export function getExtensions(type) {
     return (extensions, directory) => (initialState) => {
+        debug('Getting extensions for type \'%s\'', type);
```

Comment:
What do you think about using template string since we are transpiling all of the code anyway? 
```
debug(`Getting extensions for type ${type}`);
```